x-pack/plugins/cases/server/services/attachments/test_utils.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObject } from '@kbn/core/server';
+import { FILE_SO_TYPE } from '@kbn/files-plugin/common';
+import type {
+  AttributesTypeAlerts,
+  AttributesTypeUser,
+  CommentAttributesWithoutRefs,
+  ExternalReferenceWithoutRefs,
+} from '../../../common/api';
+import {
+  ExternalReferenceStorageType,
+  FILE_ATTACHMENT_TYPE,
+  CommentType,
+} from '../../../common/api';
+import {
+  CASE_COMMENT_SAVED_OBJECT,
+  CASE_SAVED_OBJECT,
+  SECURITY_SOLUTION_OWNER,
+} from '../../../common/constants';
+import { CASE_REF_NAME, EXTERNAL_REFERENCE_REF_NAME } from '../../common/constants';
+
+export const createErrorSO = () =>
+  ({
+    id: '1',
+    type: CASE_COMMENT_SAVED_OBJECT,
+    error: {
+      error: 'error',
+      message: 'message',
+      statusCode: 500,
+    },
+    references: [],
+    // casting because this complains about attributes not being there
+  } as unknown as SavedObject<AttributesTypeUser>);
+
+export const createUserAttachment = (attributes?: object): SavedObject<AttributesTypeUser> => {
+  return {
+    id: '1',
+    type: CASE_COMMENT_SAVED_OBJECT,
+    attributes: {
+      comment: 'Wow, good luck catching that bad meanie!',
```

Comment:
Is funny how this follows us after 3 years 😆.

Reply:
Haha, good ol' copy paste!